src/type/schema.js

```diff
@@ -254,6 +285,13 @@ export class GraphQLSchema {
     return Boolean(possibleTypeMap[abstractType.name][possibleType.name]);
   }
 
+  isImplementation(
```

Comment:
Same here, though it's used once in this diff: we could either add an internal-only utility function for this for this PR, or land a PR with this included before this.

Reply:
(Same as above.) This function actually does call `getImplementations`; I broke them into two primarily to respect the pattern used by `getPossibleTypes`/`isPossibleType`.